.husky/.gitignore

```diff
@@ -1 +1 @@
-_
+_
```

Comment:
In the original PR I added a newline to this `.gitignore` because it's generally good practice for files to end with newlines. However, every time `husky install` runs (which happens after every `yarn install`, it updates this file. Might as well merge it in this PR so it won't show up as a random diff for anyone.

Reply:
For reference since git doesn't really show this change very well on the `Conversation` tab:
![image](https://user-images.githubusercontent.com/15805554/108557845-21705380-72c7-11eb-821c-80e512277265.png)